common.mk

```diff
@@ -61,7 +61,11 @@ setup-precommit: install-uv  ##- Set up pre-commit hooks in this repository.
 ifeq ($(shell which pre-commit),)
 	uv tool install pre-commit
 endif
+ifeq ($(shell which pre-commit),)
```

Comment:
It's the same if as previous so you may as well squeeze them
```suggestion
```

Reply:
Ahh right, because those get executed before running the script. So actually I can get rid of the `uv tool install` entirely since it's implicit in the run.